docs/contributing/how-to-release.md

```diff
@@ -171,20 +172,32 @@ WINDOWS71SDK
 
 If either of these are blank they are set to the default install location.
 
+For Visual Studio 2017 and 2019 the VSxxxxCOMNTOOLS environment variable is not
+set by the installer. Each of these studio version supports multiple toolsets.
+The build script will check the following in sequence:
+ - If VSxxxCOMNTOOLS is set (xxx=150: VS2017, xxx=160: VS2019) it is used as
+   the path to call VsDevCmd.bat to set up the environmet.
+ - If VSxxxCOMNTOOLS is not set, then the Visual Studio tool vswhere is used
+   to find the latest installed toolset for the compiler and the associated
+   VsDevCmd.bat file is used to set up the environment.
+
 To build binaries for a single compiler, open a command prompt (for Visual
 Studio 2008 only an SDK 6.1 developer's command prompt must be used),
 cd to the build\tools\msvs folder and run the batch file 'officialbuild'
 with the vcXXX version number:
 
-Visual Studio 2008  vc90
-Visual Studio 2010  vc100
-Visual Studio 2012  vc110
-Visual Studio 2014  vc120
-Visual Studio 2015  vc140
+    Visual Studio 2008  vc90
+    Visual Studio 2010  vc100
+    Visual Studio 2012  vc110
+    Visual Studio 2014  vc120
+    Visual Studio 2015  vc140
+    Visual Studio 2017  vc150
```

Comment:
According to _buildall.bat_ and _officialbuild.bat_ (and elsewhere), the versions for MSVS 2017 and 2019 should be `vc141` and `vc142`?

Reply:
You are correct. I was flipping back and forth between files and picked up the "version" numbers MS uses in the vswhere command. Momentary brain freeze on my part. Will update shortly.

Reply:
BTW, did you talk with @vadz and has been the final decision made about which binaries for which MSVS versions will be officially released? vadz thought that if 2015-2019 were truly interchangeable, perhaps just one of the three would suffice, see https://github.com/wxWidgets/wxWidgets/pull/1557#issuecomment-532765654